packages/rs-dpp/src/state_transition/mod.rs

```diff
@@ -126,6 +131,10 @@ impl StateTransitionConvert for StateTransition {
     fn binary_property_paths() -> Vec<&'static str> {
         panic!("Static call is not supported")
     }
+
+    fn to_cleaned_object(&self, skip_signature: bool) -> Result<Value, ProtocolError> {
+        call_method!(self, to_cleaned_object, skip_signature)
```

Comment:
didn't know we had this macro, it frightens me

Reply:
I think it's much better if we just were to use an enum.